OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -105,6 +105,7 @@
     <Compile Include="Actor.cs" />
     <Compile Include="CacheStorage.cs" />
     <Compile Include="FileSystem\IPackage.cs" />
+    <Compile Include="Int32Matrix4x4.cs" />
```

Comment:
Can we please put this in Primitives alongside our other composite number types?

Reply:
Done.